aws-ebs-csi-driver.yaml

```
@@ -1,10 +1,16 @@
 package:
   name: aws-ebs-csi-driver
   version: 1.21.0
-  epoch: 1
+  epoch: 2
   description: CSI driver for Amazon EBS.
   copyright:
     - license: Apache-2.0
+  dependencies:
+    runtime:
+      - umount
+      - util-linux
```

```suggestion
    - mount
    - lsblk
    - blkid
```

`util-linux` is just a meta package without any of the actual bins, these seem to be the ones we need

---

okay cool, thanks for letting me know!